tendermint/src/lib.rs

```diff
@@ -13,7 +13,7 @@
     unused_qualifications
 )]
 #![forbid(unsafe_code)]
-#![doc(html_root_url = "https://docs.rs/tendermint/0.13.0")]
+#![doc(html_root_url = "https://docs.rs/tendermint/0.14.0")]
```

Comment:
We should add this to sub-crates too:
https://doc.rust-lang.org/rustdoc/the-doc-attribute.html#html_root_url